### ISSUE TITLE
Bluetooth Relay to send out captured packets by the Bluetooth Proxy

### ISSUE THREAD

**Issue by LucaDev** [esphome/feature-requests#1856]
**Describe the problem you have/What new integration you would like**

I'd love to see a Bluetooth relay which can send out the Bluetooth packets captured by the Bluetooth proxy. Instead of using them inside of HomeAssistant, this could enable users to create Bluetooth over Ethernet bridges for long distance communication.

**Please describe your use case for this integration and alternatives you've tried:**

Bluetooth signals are very limited in their range and there is currently no Bluetooth over Ethernet Adapter available to buy as far as I know of.

**Additional context**

**Comment by nagyrobi** [comment]
Yes this is indeed a good idea, but it needs some implementation on the other side too, something which can emulate a Bluetooth dongle in the software. This is done in HA, question is, what similar implementation exists outside of this.
